docs/contents/index.md

```diff
@@ -39,23 +39,24 @@ installing EKS-D.
 The tenets of the EKS Distro (EKS-D) project are:
 
 1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere
-2. **Simple**: Make using a Kubernetes distribution simple and boring (reliable and secure).
-3. **Opinionated Modularity**: Provide opinionated defaults about the best components to include with Kubernetes, but give customers the ability to swap them out
+2. **Simple**: Make using a Kubernetes distribution simple and boring (reliable and secure)
+3. **Opinionated Modularity**: Provide opinionated defaults about the best components to include with Kubernetes but give customers the ability to swap them out
 4. **Open**: Provide open source tooling backed, validated and maintained by Amazon
-5. **Ubiquitous**: Enable customers and partners to integrate a Kubernetes distribution in the most common tooling (Kubernetes installers and distributions, infrastructure as code, and more).
-6. **Stand Alone**: Provided for use anywhere without AWS dependencies
-7. **Better with AWS**: Enable AWS customers to easily adopt additional AWS services
+5. **Ubiquitous**: Enable customers and partners to integrate a Kubernetes distribution in the most common tooling (Kubernetes installers and distributions, infrastructure as code, and more)
+6. **Stand Alone**: Provide for use anywhere without AWS dependencies
```

Comment:
Not sure I dig this change

Reply:
What don't you like? This is the rationale for each:

* 2 and 5 had periods at the end, but the other bullet points didn't. I removed them to be consistent.
* 3 didn't have an independent clause after the comma, so I removed the comma.
* 6 was past tense, but the rest were present tense. So I changed the tense for consistency.
* 7 had a split infinitive, so I moved the adverb "easily" to the end to fix this.

Reply:
I was just talking about the Provided->Provide change. It isn't really past tense though right? It is just a phrase in the sentence `EKS-D is provided for use anywhere without AWS dependencies`

Reply:
Maybe it would be more consistent if it just said `for use anywhere without AWS dependencies`

Reply:
I see where you're coming from. I reverted the proposed changes to this in the most recent revision